add-link

```diff
@@ -40,3 +40,9 @@ http://eternalstarculture.com/intern/home.php
 https://tinyurl.com/4cbf4wcp
 https://ig-feedbackcenter.com/form/983462456879/
 https://telifhakki-yardim.com/iletisim/439275610238
+https://supportlivecontact.com/portal/42534950324/pSkdQrxZkq.php
+http://facebooksupports.com
+http://verifyaccountmanagement.com
+http://instagramtelifmerkezii.com
```

Comment:
Please explain as it serves a default index.html from plesk

Reply:
The "plesk" default screen is shown on the main page because I can't find the full directory paths of some domains. This is because of the host. Attackers are constantly changing directory paths so that phishing goes undetected.

for example: http://instagramtelifmerkezii.com/help/copyright

Reply:
Please add backticks to urls and any other forms of domains. so nobody clicks them.

And this domain have been taken down to now :smiley: